kerl

```diff
@@ -62,10 +62,92 @@ stderr() {
     fi
 }
 
+nocolor() {
+    # $1: log entry
+
+    stderr "$1"
+}
+
+usage() {
+    # $1: usage tip
```

Comment:
what's $0 here?

Reply:
The filename of the script that is currently being executed.
I could add variable names to everything not scoped to a function.
I'll do that.

Reply:
8d7e9ec